Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/corefinement.h

```diff
@@ -695,6 +695,19 @@ corefine_and_compute_difference(      TriangleMesh& tm1,
  *     \cgalParamDefault{`false`}
  *     \cgalParamExtra{`np1` only}
  *   \cgalParamNEnd
+ *   \cgalParamNBegin{do_not_modify}
+ *     \cgalParamDescription{if `true`, the corresponding mesh will not be updated.}
+ *     \cgalParamType{Boolean}
+ *     \cgalParamDefault{`false`}
+ *     \cgalParamExtra{If this parameter is set to `true` for both meshes nothing will be done.
+ *                      If this option is set to `true` for one mesh,
+ *                      the other mesh is no longer required to be without self-intersection.}
+ *   \cgalParamNEnd
+ *   \cgalParamBegin{do_not_modify} 
```

Comment:
`\cgalParamBegin` should be `\cgalParamNBegin` (analogous `\cgalParamEnd` -> `\cgalParamNEnd`)